euth_wagtail/assets/scss/components/_badges.scss

```diff
@@ -0,0 +1,28 @@
+.euth-badge {
```

Comment:
Was there a reason to create a new badge class instead of overriding the bootstrap `.badge` styles with it? I think, this is how we ended up with several button classes.